cli.go

```diff
@@ -361,5 +361,8 @@ func (i *CLI) amqpErrorWatcher(amqpConn *amqp.Connection) {
 	if ok {
 		i.logger.WithField("err", err).Error("amqp connection errored, terminating")
 		i.cancel()
+		time.Sleep(2 * time.Minute)
+		i.logger.Error("timed out waiting for shutdown after amqp connection error")
```

Comment:
I approve of this comment ![approved](https://theoverlookedonlookers.files.wordpress.com/2013/10/reverse-1269602956_dr-mccoy-and-captain-kirk-approve.gif)